ModernWpf/Media/Animation/PointerUpThemeAnimation.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Windows;
+using System.Windows.Media.Animation;
+using ModernWpf.Controls.Primitives;
+
+namespace ModernWpf.Media.Animation
+{
+    /// <summary>
+    /// Represents the preconfigured pointer up animation that applies to controls when
```

Comment:
Typo error : have to change 'when' to 'after' and 'pressed' to 'released' in the xml comment